app/uk/gov/hmrc/agentmappingfrontend/views/start.scala.html

```diff
@@ -53,21 +53,9 @@ <h2 class="heading-medium">@Messages("connectAgentServices.start.h2")</h2>
 
     <h2 class="heading-medium">@Messages("connectAgentServices.start.beforeYouStart")</h2>
 
-
-    @if(arnOpt.isDefined) {
-        <p class="panel panel-border-wide">@Messages("connectAgentServices.start.l2.1.dynamic-arn", arnOpt.get)</p>
```

Comment:
If these messages aren't referenced anywhere else in the project, they can be removed from the messages file..

- connectAgentServices.start.l2.1.dynamic-arn
- connectAgentServices.start.l2.1
- connectAgentServices.start.youWillAlsoNeed
- connectAgentServices.start.l2.2
- connectAgentServices.start.l2.3